CondTools/RPC/src/RPCDCCLinkMapHandler.cc

```diff
@@ -0,0 +1,116 @@
+#include "CondTools/RPC/interface/RPCDCCLinkMapHandler.h"
+
+#include <fstream>
+#include <sstream>
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include "CondCore/CondDB/interface/ConnectionPool.h"
+#include "CondCore/CondDB/interface/Types.h"
+
+#include "RelationalAccess/ICursor.h"
+#include "RelationalAccess/IQuery.h"
+#include "RelationalAccess/IQueryDefinition.h"
+#include "RelationalAccess/ISchema.h"
+#include "RelationalAccess/ISessionProxy.h"
+
+#include "CoralBase/Attribute.h"
+#include "CoralBase/AttributeList.h"
+
+#include "CondTools/RPC/interface/RPCLBLinkNameParser.h"
+
+RPCDCCLinkMapHandler::RPCDCCLinkMapHandler(edm::ParameterSet const & _config)
+    : id_(_config.getParameter<std::string>("identifier"))
+    , data_tag_(_config.getParameter<std::string>("dataTag"))
+    , since_run_(_config.getParameter<unsigned long long>("sinceRun"))
+    , txt_file_(_config.getUntrackedParameter<std::string>("txtFile", ""))
+{
+    cond::persistency::ConnectionPool _connection;
+    _connection.setParameters(_config.getParameter<edm::ParameterSet>("DBParameters"));
+    _connection.configure();
+    edm::LogInfo("RPCDCCLinkMapHandler") << "Opening Input Session";
+    input_session_ = _connection.createSession(_config.getParameter<std::string>("connect"));
```

Comment:
Can I make the ConnectionPool a class member of the Handler and configure it in the Handler constructor in this scenario?  Instinctively I would, but use within CMSSW differs.